pandas/tools/plotting.py

```diff
@@ -2555,8 +2572,10 @@ def _plot(data, x=None, y=None, subplots=False,
     figsize : a tuple (width, height) in inches
     use_index : boolean, default True
         Use index as ticks for x axis
-    title : string
-        Title to use for the plot
+    title : string or list
+        If a string is passed, print the string at the top of the figure. If a
+        list is passed and subplots is True, print each item in the
```

Comment:
single backticks (`` ` ``) around 'subplots' (to make it clear that it is a keyword)